### ISSUE TITLE
Crowdcast pros/cons

### ISSUE THREAD

**Issue by yarikoptic** [con/ference#2]
Pros
- casts to hundreds (thousands?) of participants
- has chat
  - emoticons in the chat but IIRC as a separate post (ideally better be like mattermost emoticons per post)
- makes video record of the event to share later
- every participant it's just a participant of chat without audio/video unless invited
- built in polls
- built in q&a 

Cons
- closed paid service
- limits to up to 4 active invited participants. Mitigated by starting zoom and sharing the screen. Cons: those who share can't see chat, those who participated have audio/video mismatch unless mute crowdcast (if I got it right)
- chat log, q&a, polls doesn't seem to have a permanent url
- chat is rudimentary, Afaik no `@references` (and to associate response with a particular comment) supported

Pros/cons (feature which might be needed)
- allows (or requires?) authorized (login/password) to access

**Comment by r03ert0** [comment]
It may be good to compare Crowdcast vs Jitsi, to better see which functionalities of crowdcast cannot be implemented open source: Jitsi has a chat, can record videos to youtube and vimeo, participants can have audio and video muted. Unclear if it can work for broadcasting alone. Jitsi's iframe API is a big plus (especially the data channel, which enables server-less inter-user interaction).

**Comment by r03ert0** [comment]
https://obsproject.com/ may be a good open source alternative for broadcasting.

**Comment by yarikoptic** [comment]
Here I would like to collect information about features of crowdcast we would like to have. We better discuss alternative setups in a dedicated issue https://github.com/con/ference/issues/3 to which I had added a comment on the amazing OBS which I love (I would say it is not for every participant to use)

**Comment by soichih** [comment]
My daughter uses OBS to record her video and post them on YouTube. She says it's the best screen recording tool that's out there used by many YouTubers.

**Comment by yarikoptic** [comment]
Yes, OBS is **awesome**, and could be used probably at different "levels". E.g. I initiated #15 now to collect recommendations for speakers/participants. It, and in particular  a windows-only (but might work on mac) clone mentioned in #4.
